pom.xml

```diff
@@ -56,17 +56,17 @@
         <alfresco.sdk.tests.exclude>*/*-enterprise*/*</alfresco.sdk.tests.exclude>
 
         <alfresco.bomDependency.artifactId>acs-community-packaging</alfresco.bomDependency.artifactId>
-        <alfresco.platform.version>7.2.0</alfresco.platform.version>
+        <alfresco.platform.version>7.3.0-A20</alfresco.platform.version>
         <alfresco.platform.docker.user>alfresco</alfresco.platform.docker.user>
-        <alfresco.share.docker.version>7.2.0</alfresco.share.docker.version>
+        <alfresco.share.docker.version>7.3.0-A20</alfresco.share.docker.version>
         <!--
              The following value is now obtained by looking at the
              - alfresco-community-share.version (eg. https://github.com/Alfresco/acs-community-packaging/blob/7.1.0/pom.xml#L17)
              or
              - alfresco-enterprise-share.version (eg. https://github.com/Alfresco/acs-packaging/blob/7.1.0.1/pom.xml#L18)
              for the GitHub Tag related to the above Docker version.
          -->
-        <alfresco.share.version>14.96</alfresco.share.version>
+        <alfresco.share.version>17.96</alfresco.share.version>
```

Comment:
The version of image (A20) relates to share in version 17.94. Should we use the newer one, or should we stick to version from image?